app/assets/stylesheets/tab.scss

```diff
@@ -0,0 +1,74 @@
+.tab-nav-link {
+	background-color: #fff;
+  padding: 5px 20px;
+  display: inline-block;
+	border-radius: 25px;
+}
+.tab-nav-link.active {
+	background: #f6f7f8;
+	color: #007BFF;
+}
+.tab-nav-link.active a {
+	color: #007BFF;
```

Comment:
Looks like the indentation is not right, can you fix this?

Reply:
resolved by vasanth